src/commonComponents/linkItem/LinkItem.js

```diff
@@ -0,0 +1,40 @@
+// Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+
+function renderLinkItem(props, ref) {
```

Comment:
This should be a component which should be wrapped inside React.forwardRef(......)